### PR TITLE
added samples for Serilog with code config and file config

### PR DESCRIPTION
**Pull request by SrutiG**
I added sample code for Serilog to the samples directory. 

SerilogTestCode uses code configuration for the logger and SerilogTestCodeFromConfig uses a config file (appsettings.json) to configure the logger

### CODE REVIEW

File: samples/Serilog/SerilogTestCode/Program.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            AWSLoggerConfig configuration = new AWSLoggerConfig("Serilog.ConfigExample");
+            configuration.Region = "us-east-1";
+
+            var logger = new LoggerConfiguration()
+            .WriteTo.AWSSeriLog(configuration)
```
Comment (costleya) [inline samples/Serilog/SerilogTestCode/Program.cs]: indent "." chains on new lines

```csharp
var logger = new LoggerConfiguration()
    .WriteTo.AWSSeriLog(configuration)
    .CreateLogger();
```

File: samples/Serilog/SerilogTestCode/Program.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+using AWS.Logger.SeriLog;
+using AWS.Logger;
+
+namespace SerilogTestCode
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            AWSLoggerConfig configuration = new AWSLoggerConfig("Serilog.ConfigExample");
```
Comment (costleya) [inline samples/Serilog/SerilogTestCode/Program.cs]:
```csharp
var configuration = new AWSLoggerConfig("Serilog.ConfigExample")
{
    Region = "us-east-1"
}
```

File: samples/Serilog/SerilogTestCodeFromConfig/Program.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿using Serilog;
+using Microsoft.Extensions.Configuration;
+
+namespace SerilogTestCodeFromConfig
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // logger configuration reads from appsettings.json
```
Comment (costleya) [inline samples/Serilog/SerilogTestCodeFromConfig/Program.cs]: same comments as other Program.cs

**Comment by SrutiG** [comment]
I added the indentation changes and I also updated the README with a link

**Pull request merged**